modules/tinymce/src/plugins/image/main/ts/api/Settings.ts

```diff
@@ -25,18 +26,12 @@ const hasImageCaption = (editor: Editor) => editor.getParam('image_caption', fal
 
 const getImageList = (editor: Editor) => editor.getParam('image_list', false);
 
-const hasUploadUrl = (editor: Editor) => !!getUploadUrl(editor);
+const hasUploadUrl = (editor: Editor) => Type.isNonNullable(editor.getParam('images_upload_url'));
 
 const hasUploadHandler = (editor: Editor) => !!getUploadHandler(editor);
 
-const getUploadUrl = (editor: Editor) => editor.getParam('images_upload_url', '', 'string');
-
 const getUploadHandler = (editor: Editor) => editor.getParam('images_upload_handler', undefined, 'function');
```

Comment:
This should also be removed/inlined into `hasUploadHandler` as we no longer want to "get" the upload handler, we only want to see if it's configured.

Reply:
Done